src/main/java/com/microfocus/application/automation/tools/run/AbstractRemoteRunner.java

```diff
@@ -0,0 +1,90 @@
+package com.microfocus.application.automation.tools.run;
```

Comment:
License missing and please move this class to SV folder

Reply:
Missing license is not nice. I'm going to fix it. Anyway, are you able to configure automatic checks for it?

Reply:
Do you mean adding the license checks as part of the CI?